public/favicon

```diff
@@ -0,0 +1 @@
+This file remains empty but needs to stay.
```

Comment:
This should not be here instead adding the (.ico) extension to the name of this file.